From a6d5710b0fa7d79da7addc1ba2f58b28d6823396 Mon Sep 17 00:00:00 2001
From: Michael Forney <mforney@mforney.org>
Date: Tue, 1 Jun 2021 00:24:45 -0700
Subject: [PATCH] Remove duplicate keymap sequences from st terminfo

Otherwise, curses has to make an arbitrary decision about which key
to use when that sequence is detected.
---
 share/terminfo/terminfo | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/share/terminfo/terminfo b/share/terminfo/terminfo
index b70fa1c..6e1f3bb 100644
--- a/share/terminfo/terminfo
+++ b/share/terminfo/terminfo
@@ -1398,7 +1398,7 @@ qansi-g|QNX ANSI,
 	ind=\E[S, invis=\E[9m,
 	is2=\E>\E[?1l\E[?7h\E[0;10;39;49m, is3=\E(B\E)0,
 	kBEG=\ENn, kCAN=\E[s, kCMD=\E[t, kCPY=\ENs, kCRT=\ENt,
-	kDL=\ENv, kEXT=\ENw, kFND=\ENx, kHLP=\ENy, kHOM=\E[h,
+	kEXT=\ENw, kFND=\ENx, kHLP=\ENy, kHOM=\E[h,
 	kLFT=\E[d, kNXT=\E[u, kOPT=\ENz, kPRV=\E[v, kRIT=\E[c,
 	kbs=^H, kcan=\E[S, kcbt=\E[Z, kclo=\ENc, kclr=\ENa,
 	kcmd=\E[G, kcpy=\E[g, kctab=\E[z, kcub1=\E[D, kcud1=\E[B,
@@ -6701,10 +6701,10 @@ st-0.6|simpleterm 0.6,
 	il=\E[%p1%dL, il1=\E[L, ind=\n, invis=\E[8m,
 	is2=\E[4l\E>\E[?1034l, kDC=\E[3;2~, kEND=\E[1;2F,
 	kHOM=\E[1;2H, kIC=\E[2;2~, kLFT=\E[1;2D, kNXT=\E[6;2~,
-	kPRV=\E[5;2~, kRIT=\E[1;2C, ka1=\E[1~, ka3=\E[5~, kb2=\EOu,
-	kbs=^?, kc1=\E[4~, kc3=\E[6~, kcbt=\E[Z, kclr=\E[3;5~,
+	kPRV=\E[5;2~, kRIT=\E[1;2C, kb2=\EOu,
+	kbs=^?, kcbt=\E[Z, kclr=\E[3;5~,
 	kcub1=\EOD, kcud1=\EOB, kcuf1=\EOC, kcuu1=\EOA,
-	kdch1=\E[3~, kdl1=\E[3;2~, ked=\E[1;5F, kel=\E[1;2F,
+	kdch1=\E[3~, kdl1=\E[3;2~, ked=\E[1;5F,
 	kend=\E[4~, kent=\EOM, kf1=\EOP, kf10=\E[21~, kf11=\E[23~,
 	kf12=\E[24~, kf13=\E[1;2P, kf14=\E[1;2Q, kf15=\E[1;2R,
 	kf16=\E[1;2S, kf17=\E[15;2~, kf18=\E[17;2~,
@@ -6724,7 +6724,7 @@ st-0.6|simpleterm 0.6,
 	kf61=\E[1;4P, kf62=\E[1;4Q, kf63=\E[1;4R, kf7=\E[18~,
 	kf8=\E[19~, kf9=\E[20~, khome=\E[1~, kich1=\E[2~,
 	kil1=\E[2;5~, kind=\E[1;2B, kmous=\E[M, knp=\E[6~,
-	kpp=\E[5~, kri=\E[1;2A, krmir=\E[2;2~, mc0=\E[i, mc4=\E[4i,
+	kpp=\E[5~, kri=\E[1;2A, mc0=\E[i, mc4=\E[4i,
 	mc5=\E[5i, op=\E[39;49m, rc=\E8, rev=\E[7m, ri=\EM,
 	rmacs=\E(B, rmir=\E[4l, rmkx=\E[?1l\E>, rmso=\E[27m,
 	rmul=\E[24m, rs1=\Ec, rs2=\E[4l\E>\E[?1034l, sc=\E7,
-- 
2.31.1

